packages/run-protocol/src/core-proposal.js

```diff
@@ -2,27 +2,14 @@ export * from './econ-behaviors.js';
 export * from './sim-behaviors.js';
 
 const SHARED_POST_BOOT_MANIFEST = harden({
```

Comment:
This isn't as easy to maintain, now that it's out of scope of `test-boot.js` in `packages/vats`.